server/src/main/java/io/crate/replication/logical/repository/LogicalReplicationRepository.java

```diff
@@ -359,7 +348,7 @@ private void restoreShardUsingMultiChunkTransfer(Store store,
                     LOGGER,
                     clusterService.getClusterName().value(),
                     store,
-                    RecoverySettings.INDICES_RECOVERY_MAX_CONCURRENT_FILE_CHUNKS_SETTING.get(settings),
+                    recoverySettings.getMaxConcurrentFileChunks(),
```

Comment:
I think we should use a dedicated setting here only used by logical replication. Recovery inside the same cluster usually has a higher priority than between clusters which would be reflected by different concurrency. Suggested name `replication.logical.recovery.max_concurrent_file_chunks`.
This will also remove the dependency to the `RecoverySettings`.

Reply:
Done in https://github.com/crate/crate/pull/12142/commits/2256d3379a8b90292700fe7b6d782445f821904e